### ISSUE TITLE
Ember.Set is deprecated

### ISSUE THREAD

**Issue by sajt** [emberjs/guides#558]
I tried the Ember.Set from http://guides.emberjs.com/v1.13.0/object-model/enumerables/ but I get 
```
DEPRECATION: Ember.Set is deprecated and will be removed in a future release.
```
Is this good? And what I can use instead?

**Comment by rwjblue** [comment]
That section of the guides needs to be updated (removing reference of Ember.Set). Would you mind opening an issue on github.com/emberjs/guides for that?

As far as what to use instead, if you really need a Set I'd suggest that you use an ES6 poly fill version (there are a few IIRC).

**Comment by locks** [comment]
Closed by #555
